Notebooks/GPRWidget.ipynb

```diff
@@ -172,15 +203,26 @@
     {
      "data": {
       "application/vnd.jupyter.widget-view+json": {
-       "model_id": "3e3e6d15256d4738a126aee945cc8897"
+       "model_id": "a01f4d2a6cbd4917a65f56df1a247858"
       }
      },
      "metadata": {},
      "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<function gpgLabs.GPR.GPRlab1.PipeWidgetFcn>"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
     }
    ],
    "source": [
-    "i = interact(PipeWidget, epsr = (0, 100, 1), h=(0.1, 2.0, 0.1), xc=(0., 40., 0.2), r=(0.1, 3, 0.1) )"
+    "radargramImage = \"../figures/GPR/ubc_GPRdata.png\"\n",
+    "PipeWidget(radargramImage)"
```

Comment:
@dccowan : The figure is fairly small. We can make it bigger
try fig = plt.figure(figsize = (10,5))